src/test/java/com/conveyal/gtfs/GTFSTest.java

```diff
@@ -462,8 +536,31 @@ private void assertThatImportedGtfsMeetsExpectations(
                     break;
                 }
             }
-            assertThatPersistenceExpectationRecordWasFound(numRecordsSearched, foundRecord);
+            assertThatPersistenceExpectationRecordWasFound(numRecordsSearched, foundRecord, fieldsWithMismatches);
+        }
+    }
+
+    private void assertThatValidationErrorsMatchExpected(
```

Comment:
This method is a little unclear as to what it does.  I think it could be made cleaner by renaming the method to `countValidationErrorsOfType` and pull the assertion out of it to the level where it is called.

Reply:
Was this going to be refactored?

Reply:
Yes, apologies for missing that.